demos/notebooks/demo_pipeline_cnmfE.ipynb

```diff
@@ -159,35 +248,62 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "%%time\n",
     "if motion_correct:\n",
     "    # do motion correction rigid\n",
-    "    mc = MotionCorrect(fnames, dview=dview, **opts.get_group('motion'))\n",
-    "    mc.motion_correct(save_movie=True)\n",
-    "    fname_mc = mc.fname_tot_els if pw_rigid else mc.fname_tot_rig\n",
+    "    mot_correct = MotionCorrect(movie_path, dview=cluster, **parameters.get_group('motion'))\n",
+    "    mot_correct.motion_correct(save_movie=True)\n",
+    "    fname_mc = mot_correct.fname_tot_els if pw_rigid else mot_correct.fname_tot_rig\n",
     "    if pw_rigid:\n",
-    "        bord_px = np.ceil(np.maximum(np.max(np.abs(mc.x_shifts_els)),\n",
-    "                                     np.max(np.abs(mc.y_shifts_els)))).astype(int)\n",
+    "        bord_px = np.ceil(np.maximum(np.max(np.abs(mot_correct.x_shifts_els)),\n",
+    "                                     np.max(np.abs(mot_correct.y_shifts_els)))).astype(int)\n",
     "    else:\n",
-    "        bord_px = np.ceil(np.max(np.abs(mc.shifts_rig))).astype(int)\n",
-    "        plt.subplot(1, 2, 1); plt.imshow(mc.total_template_rig)  # % plot template\n",
-    "        plt.subplot(1, 2, 2); plt.plot(mc.shifts_rig)  # % plot rigid shifts\n",
+    "        bord_px = np.ceil(np.max(np.abs(mot_correct.shifts_rig))).astype(int)\n",
+    "        # Plot shifts\n",
+    "        plt.plot(mot_correct.shifts_rig)  # % plot rigid shifts\n",
     "        plt.legend(['x shifts', 'y shifts'])\n",
     "        plt.xlabel('frames')\n",
     "        plt.ylabel('pixels')\n",
+    "        plt.gcf().set_size_inches(6,3)\n",
     "\n",
     "    bord_px = 0 if border_nan == 'copy' else bord_px\n",
     "    fname_new = cm.save_memmap(fname_mc, base_name='memmap_', order='C',\n",
     "                               border_to_0=bord_px)\n",
     "else:  # if no motion correction just memory map the file\n",
-    "    fname_new = cm.save_memmap(fnames, base_name='memmap_',\n",
+    "    fname_new = cm.save_memmap(movie_path, base_name='memmap_',\n",
     "                               order='C', border_to_0=0, dview=dview)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Load memory mapped file"
+    "Compare original (left) and motion corrected movie (right).\n",
+    "\n",
+    "You will probably notice they look pretty similar, as there wasn't much motion to begin with. You can see from the shift plot (plotted above) that the extracted shifts were all very small."
```

Comment:
You will notice they have a similar appearance, because there was little motion in the original data.